rootfs-builder/euleros/rootfs_lib.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+check_program(){
+	type "$1" >/dev/null 2>&1
+}
+
+check_root()
+{
+	if [ "$(id -u)" != "0" ]; then
+		echo "Root is needed"
+		exit 1
+	fi
+}
+
+generate_yum_config()
+{
+	cat > "${DNF_CONF}" << EOF
+[main]
+cachedir=/var/cache/euleros-osbuilder
+keepcache=0
+debuglevel=2
+logfile=/var/log/yum-euleros.log
+exactarch=1
+
+[Base]
+name=EulerOS-2.2 Base
+baseurl=http://developer.huawei.com/ict/site-euleros/euleros/repo/yum/2.2/os/x86_64/
+enabled=1
+gpgcheck=1
+gpgkey=http://developer.huawei.com/ict/site-euleros/euleros/repo/yum/2.2/os/RPM-GPG-KEY-EulerOS
```

Comment:
This looks insecure. Does that site support https? If so, please can you change all URs to that. If not, for CentOS we store the key in this repo.

/cc @jcvenegas.

Reply:
See #16.

Reply:
@jodh-intel  sounds good, I can update for CentOS

Reply:
Unluckily, we don't have 'https' support yet.  I'll store  the key in github now.